### PR TITLE
abuse.ch collectors and parsers

### PR DESCRIPTION
**Pull request by robcza**
Full list collectors and parsers for Zeus, Palevo and Feodo trackers. Does not include RSS feeds.

### CODE REVIEW

**Comment by robcza** [comment]
RSS feeds could be included separately later as discussed here https://github.com/certtools/intelmq/issues/292

File: intelmq/bots/parsers/abusech/parser_domain.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import re
```
Comment (sebix) [inline intelmq/bots/parsers/abusech/parser_domain.py]: Unused import re

**Comment by sebix** [comment]
You are missing the `bots/parsers/abusech/__init__.py`. Otherwise the submodule cannot be imported -> Parser fails to start.

**Comment by sebix** [comment]
The documentation on Harmonization does not mention a field called `source.fqdn`, only `source_domain_name` (or `source.domain_name`), see also #185 

How to handle this @aaronkaplan and @SYNchroACK?

**Comment by robcza** [comment]
thank you @sebix forgot to add ```bots/parsers/abusech/__init__.py```

File: intelmq/bots/parsers/abusech/parser_domain.py (hunk 1 above)
Comment (robcza) [inline intelmq/bots/parsers/abusech/parser_domain.py]: import removed, thank you @sebix

**Comment by aaronkaplan** [comment]
On Aug 21, 2015, at 10:34 AM, Sebastian <notifications@github.com> wrote:

> The documentation on Harmonization does not mention a field called source.fqdn, only source_domain_name (or source.domain_name), see also #185
> 

@sebix and me looked at it now. We'll go for {source,destination}.fqdn.
Why?
Because it is the most commonly used.

However, it is not documented in the Harmonisation document. I'd say we should also document this there :) Thank you!

Best,
a.

**Comment by SYNchroACK** [comment]
#294 - issue with the enhancements mentioned by @sebix

I will create a new document with the new harmonization.

Thank you @robcza for the code.

**Pull request merged**